lib/devicetrust/assert/assert_test.go

```diff
@@ -87,78 +85,83 @@ func TestCeremony(t *testing.T) {
 			}))
 			require.NoError(t, err, "NewCeremony errored")
 
-			authnStream, err := devicesClient.AuthenticateDevice(ctx)
-			require.NoError(t, err, "AuthenticateDevice errored")
-
-			// Typically this would be some other, non-DeviceTrustService stream, but
-			// here this is a good way to test it (as it runs actual fake device
-			// authn)
-			if err := assertC.Run(ctx, &assertStreamAdapter{
-				stream: authnStream,
-			}); err != nil {
-				t.Errorf("Run returned err=%q, want nil", err)
-			}
+			clientToServer := make(chan *devicepb.AssertDeviceRequest, 1)
+			serverToClient := make(chan *devicepb.AssertDeviceResponse, 1)
```

Comment:
Suggestion: drop the buffers? We don't really seem to need them.

```suggestion
			clientToServer := make(chan *devicepb.AssertDeviceRequest)
			serverToClient := make(chan *devicepb.AssertDeviceResponse)
```